### PR TITLE
AP_Baro: restore GND_TEMP reset in init()

### PR DESCRIPTION
**Pull request by robertlong13**
This was accidentally removed in 16f0df0

### CODE REVIEW

**Comment by robertlong13** [comment]
Should there be an autotest for all parameters that are supposed to reset each boot? I haven't added a test before, but this may be a good first one.

File: libraries/AP_Baro/AP_Baro.cpp [hunk 1]
```diff
@@ -587,6 +587,12 @@ void AP_Baro::init(void)
 {
     init_done = true;
 
+    // ensure that there isn't a previous ground temperature saved
+    if (!is_zero(_user_ground_temperature)) {
```
Comment (tridge) [inline libraries/AP_Baro/AP_Baro.cpp:591]: we want to allow users to set the temperature and have it persist

**Review by tridge — CHANGES_REQUESTED** [review]

**Comment by tridge** [comment]
it was a deliberate change to persist this value between reboots

File: libraries/AP_Baro/AP_Baro.cpp (hunk 1 above)
Comment (tridge) [reply libraries/AP_Baro/AP_Baro.cpp:591]: we need to fix the param docs to remove this:
```
This parameter is not persistent and will be reset to 0 every time the vehicle is rebooted
```

**Review by tridge — CHANGES_REQUESTED** [review]

**Comment by robertlong13** [comment]
Ah, I made the mistake of only looking at the commit, and forgot to look for an associated pull request to see justification for that.

Makes sense.

**Pull request closed without merging**